p2p/exchange_metrics.go

```diff
@@ -31,17 +30,16 @@ type exchangeMetrics struct {
 	responseSizeInst    metric.Int64Histogram
 	responseTimeInst    metric.Float64Histogram
 
+	clientReg metric.Registration
```

Comment:
uber minor nit: put at the bottom or the top of the struct

Reply:
done. thanks 🙏